### ISSUE TITLE
Abovehead Chat indicator sometimes turns back on, even if preference is off.

### ISSUE THREAD

**Issue by gitbirb** [cmss13-devs/cmss13#3215]
### Testmerges

#3073: 'Squad Team Leader' by morrowwolf at commit 55a7eaf7ac #3088: 'Fixes offset issue for multithreading test' by LynxSolstice at commit 95d6c01a9c #2652: 'Say\Hearing performance 515 edition' by Git-Nivrak at commit 0c8a0b70a3 #3179: 'Dancer buff + Bug Fixes' by Git-Nivrak at commit 3985fd08a6 #3185: 'GAU buff. ' by Tisx1 at commit c75cbb5947 #3192: 'discord certification' by harryob at commit 46c90b2253

### Description of the bug

The abovehead chat indicator sometimes turns "on", this started happening around when TGUI chat got merged I believe. The bandaid fix for this is turning it back on and off again, which means this is clearly a bug.

### What's the difference with what should have happened?

Abovehead Chat Indicator preference should not reenable when disabled.

### How do we reproduce this bug?

1. Not sure what causes it. 

- Sometimes it happens when you die, 
- sometimes it happens when you join the game. 

All i know is it happens a LOT
...

### Issue Bingo

- [X] Issue could be reproduced at least once
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] Couldn't find an existing issue about this (https://github.com/cmss13-devs/cmss13/issues)

**Comment by gitbirb** [comment]
Yeah figured it out a bit // If you reconnect using this button, it almost always turns back on abovehead chat indicators (at least for me)
![image](https://github.com/cmss13-devs/cmss13/assets/44538569/b327b0ea-72ca-4321-a9d6-cb51facbea6b)

**Comment by Huffie56** [comment]
still happening

**Comment by nuklearcellphoneg** [comment]
I've had it happen w/o needing to reconnect afaik(MIGHT have been caused by remaining in-game across rounds?)

Almost feels map dependent but I haven't paid attention
